### PR TITLE
Changes to include Set Operations

### PR DESCRIPTION
**Pull request by DimpleK26**
- Intersect
- Minus
- Except

- Unit tests for Intersect, Minus, Except Operations
- Fix for existing issues in Union unit tests

https://github.com/kayak/pypika/issues/250

### CODE REVIEW

File: pypika/enums.py [hunk 1]
```diff
@@ -58,9 +58,12 @@ class JoinType(Enum):
     cross = "CROSS"
 
 
-class UnionType(Enum):
-    distinct = ""
-    all = " ALL"
+class SetOperations(Enum):
```
Comment (gl3nn) [inline pypika/enums.py:61]: To be consistent with other Enums it probably makes sense to rename this to the singular SetOperation.

Comment (DimpleKannadasan) [reply pypika/enums.py:61]: will do

Comment (DimpleKannadasan) [reply pypika/enums.py]: @gl3nn Resolved the above comment

**Comment by DimpleKannadasan** [comment]
@gl3nn @twheys Could this Pr be reviewed please. I have resolved the comments

**Pull request merged**